pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3259,17 +3260,17 @@ func (c *Cloud) getInstancesByNodeNamesCached(nodeNames sets.String) ([]*ec2.Ins
 	return instances, nil
 }
 
-func (c *Cloud) getInstancesByNodeNames(nodeNames []string) ([]*ec2.Instance, error) {
+func (c *Cloud) getInstancesByNodeNames(nodeNames []string, states ...string) ([]*ec2.Instance, error) {
 	names := aws.StringSlice(nodeNames)
```

Comment:
@justinsb - is there any reason we don't include shutdown instances in this code? should we consider changing the function interface?